build-setup/macos_cross_compiled.md

```diff
@@ -27,7 +27,7 @@ On the fully configured macOS build server, run the following commands:
 
 ```
 cd /usr
-tar jcvf ~/usr-x86_64-apple-macosx10.14.tar.bz2 include lib local
+tar jcvf ~/usr-x86_64-apple-macosx10.14.tar.bz2 lib local
```

Comment:
I noticed that `include` was redundant as Apple no longer puts any headers in `/usr/include`.